localnet/grafana-dashboards/cosmos_sdk_insights.json

```diff
@@ -0,0 +1,1318 @@
+{
```

Comment:
FYI: I'm skipping reviewing the .json files

Reply:
That's fine, you will see the result next time you open grafana on localnet.